1.	# metabolomics				
2.	Tools	Implementation	Ease of use	Software availability	
3.	GC-MS 				
4.	CorrectOverloadedPeaks	R	difficult	https://cran.r-project.org/package=CorrectOverloadedPeaks	
5.	eRah	R	difficult	http://cran.r-project.org/package=erah	
6.	ADAP-GC 3.0	R	difficult	https://drive.google.com/drive/folders/0B8m21OI0wOSmeWFzZ3JxUVg0bms
7.	GC–MS SPA	Manual	medium	NA	
8.	flagme	R	difficult	https://bioconductor.org/packages/release/bioc/html/flagme.html	
9.	AnalyzeIMS	MATLAB	difficult	https://github.com/BioMEMS/AnalyzeIMS	
10.	MetabQ	R	difficult	http://metabolomics.auckland.ac.nz/index.php/projects/26	
11.	PYQUAN	Python	difficult	https://github.com/lycopoda/Pyquan	
12.	2D GC-MS				
13.	GC2MS	Webserver	easy	http://gc2ms.web.cmdm.tw/index.php/research	
14.	msPeakG	R	difficult	http://mrr.sourceforge.net/download.html	
15.	osd	R	difficult	http://metabolomicsplatform.com/applications/	
16.	SpectralAnalysis	Matlab	difficult	https://github.com/AlanRace/SpectralAnalysis	
17.	NMR				
18.	NMRPro	Web-based	easy	http://mamitsukalab.org/tools/nmrpro/	
19.	SpiNCouple	Web-based	easy	http://emar.riken.jp/spincpl	
20.	ChemSMP 	Python	difficult	http://nrc.iisc.ernet.in/hsa/gft.htm	
21.	PROMEB	Python	difficult	http://nrc.iisc.ernet.in/hsa/gft.htm	
22.	SpeckTackle	JavaScript	easy	https://bitbucket.org/sbeisken/specktackle	
23.	NMRmix	Python	difficult	http://www.nmrfam.wisc.edu/software	
24.	jsNMR	JavaScript	easy	http://nmr.au.dk/software/jsnmr/	
25.	Targeted				
26.	RIPPER	Java	medium	http://license.umn.edu/technologies/20120293_mass-spectrometry-software-improves-data-evaluation
27.	MRMAnalyzer	R	difficult	http://www.metabolomics-shanghai.org/softwaredetail.php?id=34	
28.	MetDIA	R	difficult	http://www.metabolomics-shanghai.org/softwaredetail.php?id=40	
29.	Untargeted				
30.	i-MetQ	.Net 	easy	http://ms.iis.sinica.edu.tw/comics/Software_iMet-Q	
31.	NINA	Windows	medium	http://www.sciencedirect.com/science/article/pii/S0731708516304381	
32.	MSCombine	R	difficult	https://cran.r-project.org/web/packages/MScombine/index.html	
33.	Isotopes				
34.	IsotopicLabelling 	R	difficult	https://github.com/RuggeroFerrazza/IsotopicLabelling	
35.	MID Max	Python	difficult	https://github.com/dmccloskey/MDV_utilities	
36.	geoRge	R	difficult	https://github.com/jcapelladesto/geoRge	
37.	MIA	C++	difficult	http://massisotopolomeanalyzer.lu/	
38.	ICT	Perl	difficult	https://github.com/jungreuc/isotope_correction_toolbox/	
39.	QC				
40.	LacaScore	NA	na	http://link.springer.com/article/10.1007/s11306-016-1038-1	
41.	QCScreen	Windows, Python, R	difficult	https://sourceforge.net/projects/qcscreen/	
42.	LC-MS pre-processing				
43.	MET-XAlign	C++, Microsoft.NET Framework	medium	http://bioinfo.noble.org/manuscript-support/met-xalign/	
44.	CCSWA	R	difficult	NA	
45.	MetNormalizer	R	easy	http://www.metabolomics-shanghai.org/	
46.	MetTailor	R	difficult	https://sourceforge.net/projects/mettailor/	
47.	Gross and systematic error correction	NA	na	http://pubs.acs.org/doi/abs/10.1021/acs.analchem.5b03912	
48.	GridMass	Java	easy	http://bioinformatica.mty.itesm.mx/GridMass	
49.	BatchCorrMetabolomics	R	difficult	https://github.com/rwehrens/BatchCorrMetabolomics	
50.	Annotation				
51.	1-SToP	C#, R 	difficult	http://pubs.acs.org/doi/abs/10.1021/acs.analchem.6b02479	
52.	iMet	Webserver	easy 	http://imet.seeslab.net./ 	
53.	InterpretMSSpectrum	R	difficult	https://cran.r-project.org/package=InterpretMSSpectrum	
54.	Elder	R	difficult	http://https//sourceforge.net/projects/metaopen/files/EIder/	
55.	MSFinder	C#	easy	http://prime.psc.riken.jp/Metabolomics_Software/MS-FINDER/index.html 
56.	MetFamily	R, Webserver	easy	http://msbi.ipb-halle.de/MetFamily/	
57.	Greazy	.Net, R	difficult	http://tiny.cc/bumbershoot-vc12-bin64 	
58.	tMICE	NA	na	NA	
59.	Library/ Database				
60.	WEIZMASS 	R	difficult	NA	
61.	Secondary Metabolite Bioinformatics Portal	Web-based	easy	http://www.secondarymetabolites.org/	
62.	MINEs	Webserver	easy	http://minedatabase.mcs.anl.gov/#/home	
63.	MetCCS	Webserver	easy	http://www.metabolomics-shanghai.org/softwaredetail.php?id=108	
64.	GlycoMob	Web-based	easy	http://www.glycomob.org/	
65.	XMetDB	Webserver	easy	http://www.xmetdb.org/xmetdb	
66.	Phenol-Explorer	Web-based	easy	NA	
67.	Statistical				
68.	SMART	R	difficult	http://www.stat.sinica.edu.tw/hsinchou/metabolomics/SMART.htm	
69.	RANSAC-PLS	R	difficult	http://www.sciencedirect.com/science/article/pii/S1389172316000098	
70.	tmod	R	difficult	https://cran.r-project.org/web/packages/tmod/index.html 	
71.	metaCCA	MATLAB	difficult	https://github.com/aalto-ics-kepaco 	
72.	metICA	R	difficult	https://github.com/daniellyz/MetICA 	
73.	Cardinal	R	difficult	http://cardinalmsi.org/	
74.	Pathways				
75.	Subpathway-CorSP	R	difficult	NA	
76.	KeyPathwayMiner	Web-based	easy	http://tomcat.compbio.sdu.dk/keypathwayminer/	
77.	PathVisioRPC	Java	medium	http://projects.bigcat.unimaas.nl/pathvisiorpc/	
78.	WikiPathways	R	difficult	https://github.com/egonw/rwikipathways	
79.	PaxtoolsR	R	difficult	http://www.bioconductor.org/packages/paxtoolsr	
80.	Escher	Web	easy	https://escher.github.io/	
81.	Network				
82.	MetaMapR	R, Web-based	easy	http://dgrapov.github.io/MetaMapR/	
83.	GAM	Web-based	easy	https://artyomovlab.wustl.edu/shiny/gam/	
84.	DiffCorr	R	difficult	https://cran.r-project.org/web/packages/DiffCorr/index.html	
85.	Visualization				
86.	ili	R	difficult	https://github.com/ili-toolbox/ili	
87.	msIQUANT	Windows	medium	http://ms-imaging.org/wp/paquan/	
88.	BatMass	Windows	easy	http://www.batmass.org/	
89.	Heatmapper	Webserver	easy	http://www.heatmapper.ca/	
90.	MSIdV	Python	difficult	https://sourceforge.net/projects/msidv/	
91.	Global				
92.	MAIT	R	difficult	https://bioconductor.org/packages/release/bioc/html/MAIT.html	
93.	MetaboAnalyst 3.0	Web, R	easy	http://www.metaboanalyst.ca/	
94.	Galaxy-M	MATLAB, Galaxy	difficult	https://github.com/Viant-Metabolomics/Galaxy-M	
95.	PIUMet	Webserver	easy	http://fraenkel-nsf.csbi.mit.edu/PIUMet/	
96.	specmine	R	difficult	https://cran.r-project.org/web/packages/specmine/index.html	
97.	OpenMS	Windows	easy	http://www.openms.de/	
98.	Others				
99.	ChemDataExtractor	Webserver	easy	http://chemdataextractor.org/	
100.	ConceptMetab	Webserver	medium	http://conceptmetab.med.umich.edu/	
101.	ReconMap	Webserver	easy	https://github.com/opencobra/cobratoolbox	
102.	MBROLE 2.0	Webserver	easy	http://csbg.cnb.csic.es/mbrole2	
103.	biosigner	R	difficult	https://bioconductor.org/packages/release/bioc/html/biosigner.html	
104.	USR-Vis	Webserver	easy	http://usr.marseille.inserm.fr/	
105.	CypRules	Webserver	easy	http://cyprules.cmdm.tw/	
106.	LOBSTAHS	R	difficult	https://github.com/vanmooylipidomics/LOBSTAHS	
107.	PredRet	Webserver	easy	http://predret.org/	
108.					
109.	Name		Platform Dependency	Web	References
110.	Data handling and preprocessing tools			
111.	MetMSLine		LC-MS; R	http://wmbedmands.github.io/MetMSLine/ 	
112.	MSPrep		MS, R	http://sourceforge.net/projects/msprep/	
113.	MRM-DIFF		LC-MS; Windows	http://prime.psc.riken.jp/	
114.	MRMPROBS		LC-MS; Windows	http://prime.psc.riken.jp/	
115.	FragPred		MS; R	http://pattilab.wustl.edu/software/FragPred/index.php 	
116.	IsoMS		LC-MS; R, Windows	www.mycompoundid.org/IsoMS	
117.	MyCompoundID.org		MS; Web	http://mycompoundid.org/	
118.	isoMETLIN		MS/MS; Web	http://isometlin.scripps.edu/	
119.	IPO		LC-MS; R	https://github.com/glibiseller/IPO	
120.	Massifquant/ Optimize-it		MS; Bioconductor	https://github.com/topherconley/optimize-it 	
121.	IsoMS Quant		LC-MS; R, Windows	www.mycompoundid.org/IsoMS	
122.	MUSCLE		LC-MS; C++	http://www.muscleproject.org/	
123.	intCor		LC-MS; R	http://b2slab.upc.edu/software-and-downloads/intensity-drift-correction/
124.	Peak-group-alignment		LC-MS, R	https://github.com/joewandy/peak-grouping-alignment	
125.	Parametric time warping		LC-DAD, LC-MS; R	https://github.com/rwehrens/ptw 	
126.	PeakANOVA		LC-MS; R	http://research.ics.aalto.fi/mi/software/peakANOVA/ 	
127.					
128.	Statistical tools				
129.	DeviumWeb		Any; R, Web	https://github.com/dgrapov/DeviumWeb 	
130.	EigenMS		MS; R, MATLAB	http://sourceforge.net/projects/eigenms/	
131.	RepExplore		Any; Web	http://www.repexplore.tk	
132.	Normalyzer		Any; Web	http://quantitativeproteomics.org/normalyzer	
133.	BioStatFlow		Any; Web	http://biostatflow.org/	
134.	Annotation tools				
135.	MS-DIAL		LC-MS; Windows	http://prime.psc.riken.jp/	
136.	MET-COFEA		LC-MS, Windows	http://bioinfo.noble.org/ manuscript-support/met-cofea/	
137.	MetAssign		LC-MS; Java	http://mzmatch.sourceforge.net/MetAssign.php 	
138.	MS2Analyzer		MS/MS; Java	http://fiehnlab.ucdavis.edu/projects/MS2Analyzer/	
139.	Metabolome searcher		-, Web	http://procyc.westcent.usu.edu/cgi-bin/MetaboSearcher.cgi	
140.	MAGMa		MS/MS; Java	https://www.emetabolomics.org/ 	
141.	mzGroupAnalyzer		LC-MS; MATLAB	http://www.univie.ac.at/mosys/software.html	
142.	Cosmiq		LC-MS, GC-MS; R	http://www.bioconductor.org/packages/release/bioc/html/ cosmiq.html	
143.	Feature Credential		MS/MS; R	http://pattilab.wustl.edu/software/credential/	
144.	ProbMetab		LC-MS; R	http://labpib.fmrp.usp.br/methods/probmetab/	
145.	HR3		GC-MS, LC-MS; Windows	www.metalign.nl	
146.	MAIT 		MS, R	http://b2slab.upc.edu/software-and-downloads/metabolite-automatic-identification-toolkit/ 
147.	BioSM		MS/MS; Java	http://metabolomics.pharm.uconn.edu	
148.	BioSMXpress		-	http://engr.uconn.edu/~rajasek/BioSMXpress.zip	
149.	CFM-ID		ESI-MS/MS; Web	http://cfmid.wishartlab.com/	
150.	MIDAS		MS/MS; Web	http://midas.omicsbio.org	
151.	GS-align		-; C++	http://www.glycanstructure.org/gsalign	
152.	POCHEMON		LC-MS; MATLAB	http://www.ru.nl/science/analyticalchemistry/research/software/	
153.	RAMClustR		MS/MS; R	https://github.com/cbroeckl/RAMClustR	
154.	LipidPro		MS/MS; Windows	http://www.neurogenetics. biozentrum.uni-wuerzburg.de/en/project/services/l ipidpro/
155.	mzCloud		MS/MS; Web	https://www.mzcloud.org/ 	
156.	Pathway and networks analysis and biological interpretation tools		
157.	PathCaseMAW		Any; Web	http://nashua.case.edu/PathwaysMAW/Web	
158.	MINE		-; Java, Perl, Python	http://minedatabase.mcs.anl.gov/#/home 	
159.	PathWhiz		-	http://smpdb.ca/pathwhiz	
160.	TrackSM		MS/MS; MATLAB	http://metabolomics.pharm.uconn.edu/?q=Software.html	
161.	MarVis-Suite		Any, Java	http://marvis.gobics.de	
162.	InCroMAP		Any; Java	http://www.cogsys.cs.uni-tuebingen.de/software/InCroMAP	
163.	iPEAP		Any; Windows; Java’; R	http://www.tongji.edu.cn/~qiliu/ipeap.html 	
164.	kpath		Any; Web	http://browser.kpath.khaos.uma.es/	
165.	Pathomx		Any; Python; MATLAB; R	http://pathomx.org/	
166.	MetaMapR		Any; R, Web	http://dgrapov.github.io/MetaMapR/	
167.	Metabnet		LC-MS; R	https://sourceforge.net/projects/metabnet/ 	
168.	MetaNET		Any; Galaxy	http://metanet.osdd.net	
169.	Funrich		-, Web	http://www.funrich.org/ 	
170.	Network portal		Any; Web	http://networks.systemsbiology.net	
171.	SimIndex		Any; Python	http://tyolab.northwestern.edu/tools/ 	
172.	BiNChE		-	http://www.ebi.ac.uk/chebi/tools/binche/	
173.	PlantSEED		-	http://bioseed.mcs.anl.gov/~seaver/FIG/seedviewer.cgi?page=PlantSEED	
174.	SPICA		LC-MS	http://cmcr.columbia.edu/metabolomics /informaticstools.html	
175.	KiMoSys		Any; Web	http://kimosys.or	
176.	Integrated Interactome System	Any; Web	http://www.lge.ibi.unicamp.br/lnbio/IIS/	
177.	MetaDB		GC-MS, LC-MS, R	https://github.com/rmylonas/MetaDB 	
178.	NMR-based Tools				
179.	Focus		NMR; MATLAB	http://www.urr.cat/FOCUS	
180.	Metabnorm		NMR; R	http://sourceforge.net/projects/metabnorm/ 	
181.	BATMAN		NMR; R	http://batman.r-forge.r-project.org/	
182.	Bayesil		NMR Web	http://bayesil.ca/	
183.	SENECA		NMR; Java	http://sourceforge.net/projects/seneca/	
184.	COLMAR		NMR; Web	http://spin.ccic.ohio-state.edu/index.php/colmar	
185.	1H(13C)-TOCCATA		NMR; Web	http://spin.ccic.ohio-state.edu/index.php/toccata2/index	
186.	mQTL.NMR		NMR; R	http://www.ican-institute.org/tools/	
187.	MVAPACK		NMR; MATLAB	http://bionmr.unl.edu/mvapack.php	
188.	ChemoSpec		NMR; R	http://cran.r-project.org/web/packages/ChemoSpec/ 	
189.	GC-MS-based tools				
190.	MetaMS		GC-MS; R	http://www.bioconductor.org/packages/release/bioc/html/metaMS.html 
191.	Maui-VIA		GC-MS; Java	http://bimsbstatic.mdc-berlin.de/kempa/software/kempaSoftware.html	
192.	PScore		GC-MS; R	http://raphaelaggio.github.io/	
193.	BiPACE 2D		2D GC-MS; 	http://maltcms.sf.net/ 	
194.	Library, databases and others			
195.	HAMMER		MS/MS; Java, Python	http://www.biosciences-labs.bham.ac.uk/viant/hammer/)	
196.	TAPIR		MS/MS; Python	https://github.com/msproteomicstools/msproteomicstools	
197.	TMDB		-	http://pcsb.ahau.edu.cn:8080/TCDB/	
198.	BioPhytMol database		-	http://ab-openlab.csir.res.in/biophytmol/	
199.	EssOilDB		-	http://nipgr.res.in/Essoildb/	
200.	mVOCs		-	http://bioinformatics.charite.de/mvoc	
201.	domdb		-, SQL	https://github.com/joefutrelle/domdb 	
202.	PhenoMeter		-, Web	https://www.metabolome-express.org/phenometer.php 	
203.	T3DB		-, Web	www.t3db.ca 	
204.	SwissLipids		MS, Web	http://www.swisslipids.org/	
205.	Metabolonote			http://metabolonote.kazusa.or.jp/	
206.	KOMICS		-	http://www.kazusa.or.jp/komics/	
207.	QTREDS		-; Ruby, Rails	http://qtreds.crs4.it 	
208.	MASTR-MS		-	https://mastr-ms.readthedocs.org/en/latest/	
209.	Yabi			http://ccg.murdoch.edu.au/yabi/	
210.	jmzTab		-	http://mztab.googlecode.com	
211.	PolySearch2		-; Web	http://polysearch.ca 	
212.	SpeckTackle		MS/MS, NMR; Java	https://bitbucket.org/sbeisken/specktackle 	
213.	Metabolite Imager		-, Web	www.metaboliteimager.com	
214.	EXIMS		MALDI-MS, -	http://exims.sourceforge.net/	
215.	BioMet Toolbox 2.0		-, Web	http://biomet-toolbox.org/	
216.	Multifunctional tools				
217.	XCMS Online		Web	https://xcmsonline.scripps.edu/	
218.	Mass++		Any, Java	http://www.first-ms3d.jp/english/ 	
219.	MASSyPup		LC-MS, GC-MS; Linux	http://www.bioprocess.org/massypup	
220.	MetaboNexus		Any; Windows, Java	http://www.sph.nus.edu.sg/index.php/ research-services/research-centres/ceohr/metabonexus
221.	MetaboLyzer		LC-MS; Python	https://sites.google.com/a/georgetown.edu/fornace-lab-informatics/home/ metabolyzer
222.	Workflow4Metabolomics		LC-MS, GC-MS, NMR; Galaxy	http://workflow4metabolomics.org	
223.	Haystack		LC-MS; Web	http://binf-app.host.ualr.edu/haystack/	
224.	ALLocator		LC-MS,13C; Web	https://allocator.cebitec.uni-bielefeld.de

225.	MeKO		GC-MS; Web	http://prime.psc.riken.jp/meko/	
226.	MassCascade		LC-MS, KNIME	https://bitbucket.org/sbeisken/masscascadeknime/wiki/Home	
227.					
228.					
229.					
